kubectl-toolbox/kubectl-toolbox

```diff
@@ -65,7 +67,7 @@ parse_args() {
              ;;
          c)  TOOLBOX_ENTRY_COMMAND=$OPTARG
              ;;
-         n)  TOOLBOX_NAME=$OPTARG
+         n)  KUBECTL_ARGS=" --namespace=$OPTARG"
```

Comment:
I would set this after the switch statement and just set a bash var "NAMESPACE" here. That way you can easily add more KUBECTL_ARGS later on without many code changes.

Reply:
I forgot but I need to append on it otherwise we can't add more. If we have a variable NAMESPACE then I need to manage when it's empty or not. With this generic way I don't need to manage when it's empty or not.